aiida_widgets/structures.py

```diff
@@ -21,31 +21,28 @@ def __init__(self, text="Upload Structure", node_class=None, **kwargs):
         """
 
         self.file_upload = FileUploadWidget(text)
-        self.output = ipw.Output()
```

Comment:
Can you maybe explain a bit the reasoning behind removing the output?
E.g. if we want to clear at some point what we have written, is it still possible?

Reply:
Yes, the Output was appearing above the structure, which does not look nice. All the other warning or error message were appearing below the structure